src/components/chat/chat-panel.tsx

```diff
@@ -75,10 +77,26 @@ export const ChatPanel: React.FC<IProps> = ({ user, activeNavTab, focusDocument,
       };
       logCommentEvent(eventPayload);
     }
-    return commentsPath
-      ? deleteCommentMutation.mutate(`${commentsPath}/${commentId}`)
-      : undefined;
-  }, [commentsPath, deleteCommentMutation, focusDocument, focusTileId]);
+
+    // Check if the comment exists at the legacy path
+    const primaryDocRef = firestore[0].doc(`${commentsPath}/${commentId}`);
+    const primaryDocSnap = await primaryDocRef.get();
+
+    if (primaryDocSnap.exists) {
+      return deleteCommentMutation.mutate(`${commentsPath}/${commentId}`);
+    } else {
+      // Try the alternate (simplified) path
+      const altCommentsPath = `${getSimpleDocumentPath(focusDocument || "")}/comments`;
```

Comment:
Falling back to `""` doesn't seem good here. I'd suggest doing a console.warn in that case and then returning without doing anything.